tests/test_string_utils.py

```diff
@@ -1,14 +1,41 @@
-"""Tests for RCA utilities."""
+"""Tests for utilities related to strings."""
 
 import pandas as pd
 
-from chaos_genius.core.rca.rca_utils.string_helpers import (
+from chaos_genius.core.utils.string_helpers import (
     convert_df_dims_to_query_strings,
     convert_query_string_to_user_string,
+    get_query_from_col_and_value,
+    get_query_from_cols_and_values
 )
 
 # TODO: Add tests for binned values as well
 
+def test_get_query_from_col_and_value():
+    """Test function to get query string."""
+    values = [
+        (("a", "b"), "`a`==\"b\""),
+        (("\"a\"", "\"b\""), "`\\\"a\\\"`==\"\\\"b\\\"\"")
+    ]
+
+    for (col, val), output in values:
+        assert get_query_from_col_and_value(col, val) == output
+
+
+def test_get_query_from_cols_and_values():
+    """Test function to get query strings."""
+    dimensions = ["a", "\"a\""]
+    subgroups = [
+        "b",
+        ("c", "\"c\"")
```

Comment:
Also, it would be cleaner to use single quotes for these strings:
```suggestion
        ("c", '"c"')
```